src/main/java/core/basesyntax/service/impl/TransactionServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ContentReader;
+import core.basesyntax.service.ContentWriter;
+import core.basesyntax.service.ParseTransaction;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.TransactionService;
+import core.basesyntax.service.strategy.TypeActivityStrategy;
+
+public class TransactionServiceImpl implements TransactionService {
```

Comment:
let's transfer all this logic to `Main`, having class besides `Main` that depends on many other classes is bad

Reply:
ok
let's do it